modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/util/LayoutCopyHelperImpl.java

```diff
@@ -309,18 +299,17 @@ private void _copyLayoutPageTemplateStructure(
 						sourceLayout.getGroupId(), sourceLayout.getPlid());
 		}
 
-		List<FragmentEntryLink> fragmentEntryLinks =
-			_fragmentEntryLinkLocalService.
-				getFragmentEntryLinksBySegmentsExperienceId(
-					sourceLayout.getGroupId(), segmentsExperiencesIds,
-					sourceLayout.getPlid());
+		Map<Long, FragmentEntryLink> fragmentEntryLinksMap = new HashMap<>();
```

Comment:
@JanisZhang Rename this to `fragmentEntryLinks` since the list with this name is inlined.

Reply:
@JanisZhang Sorry, it seems `fragmentEntryLinksMap` is a correct name -- `valueTypePlural + Map`.